windows_containers/configuring-byoh-windows-instance.adoc

```diff
@@ -0,0 +1,65 @@
+[id="configuring-byoh-windows-instance"]
+= Configuring Bring-Your-Own-Host (BYOH) Windows instances
```

Comment:
See my later comments. I think this title is your first procedure title and possibly not the assembly title, which could instead be something more general, such as "Using BYOH Windows instances as nodes".